src/main/java/MealDemo/service/MealPlanningServiceImpl.java

```diff
@@ -57,30 +63,24 @@ public void saveNewMeal(Meal meal) {
         System.out.println("meal saved! ");
     }
 
-
-    // DELETE
     @Override
     public boolean deleteMeal(int mealId) {
-        boolean isDeleted;
-        // find
-        Optional<Meal> tempMeal = findMealById(mealId);
+
+       Optional<Meal> tempMeal = mealRepository.findById(mealId);
```

Comment:
Hey nice! I missed this! Good change this should indeed ideally be using the Repository. Good change.